onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -300,9 +300,10 @@ public void onInfoWindowClick(Marker marker) {
         ObaTripStatus status = mMarkerData.getStatusFromMarker(marker);
 
         if (mController != null && mController.getFocusedStopId() != null) {
-            TripDetailsActivity.start(mActivity, status.getActiveTripId(), mController.getFocusedStopId());
+            TripDetailsActivity.start(mActivity, status.getActiveTripId(),
+                    mController.getFocusedStopId(), "vehicle");
```

Comment:
We'll also need to add a `SCROLL_MODE_STOP` in `TripDetailsListFragment`, and in `ArrivalsListFragment.goToTripDetails()` set the stop mode.